app/controllers/application_controller.rb

```diff
@@ -1,3 +1,3 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery
+  protect_from_forgery with: :exception
```

Comment:
In `transition`, we [rescue the exception](https://github.com/alphagov/transition/blob/5e6a4783e4864fd28be62abe3d33d26d44885f39/app/controllers/application_controller.rb#L13-L18) that is raised here so that it doesn't get logged to Sentry.

Would it be possible to do the same in this app?  Otherwise we run the risk of Sentry reporting things we can't action.

Reply:
Done :)